src/Element/AddThisWrapper.php

```diff
@@ -37,18 +37,16 @@ public function getInfo() {
    * @return mixed
    */
   public static function preRenderAddThisWrapper($element) {
```

Comment:
I know this wasn't the goal of this PR, but I'd suggest we move all this logic to a Twig template while we are changing things here.